content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -24,15 +24,32 @@ This provides observability for libraries not covered by Datadog SDKs without ch
 
 Before adding OpenTelemetry instrumentation libraries, set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
 
+Datadog SDKs provide an implementation of the OpenTelemetry API that submits spans to a Datadog Agent. To use the OpenTelemetry
```

Comment:
I think the Prerequisites section is getting overloaded with alert boxes, making it hard to read. I'd suggest replacing the entire Prerequisites section with something like this instead:
```
## Prerequisites

1. **Enable OpenTelemetry support**: Set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.

   <div class="alert alert-info">This step isn't required for the Datadog Go and Ruby APM SDKs.</div>

1. **Run the Datadog Agent**: Datadog SDKs provide an implementation of the OpenTelemetry API and submit spans to a Datadog Agent. Ensure the Datadog Agent is [running][16] to use OpenTelemetry instrumentation with Datadog SDKs.

1. **Disable duplicate instrumentation**: When replacing a Datadog instrumentation with its OpenTelemetry equivalent, disable the
Datadog instrumentation to prevent duplicate spans from appearing in the trace.
```

Then some of the unsupported functionality/limitations can be moved to the **Language support** section (see next comment).